### PR TITLE
Feature/meo 233 filter property

### PR DESCRIPTION
**Pull request by mloeffle**
- Changes filter display from accordions to popovers on desktop & offcanvas on mobile
- displays filter badges of selected Filters
- Filters can be removed through badges
- add property filters
- add badge on filter showing the sum of selected filters
- extends LayoutSidebar to also move from bottom to top (previously only left to right or right to left were supported)

### CODE REVIEW

File: components/ProductListing/Offcanvas/ProductListingFiltersOffcanvas.vue [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<script setup lang="ts">
+import type { Schemas } from '@shopware/api-client/api-types';
+import type { ListingFilter } from '../../../types/listing/filter';
+import ProductListingFiltersOffcanvasItem from '~/components/ProductListing/Offcanvas/ProductListingFiltersOffcanvasItem.vue';
```
Comment (m-borgmann) [inline components/ProductListing/Offcanvas/ProductListingFiltersOffcanvas.vue:4]: Naming of the component is not correct here, thus the auto import is not working.

Should be ProductListingOffcanvasFilterItem. Also applies to this file aswell.

File: components/ProductListing/Filter/Options/ProductListingFilterOptionsProperties.vue [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+const emits = defineEmits<{
+    'filter-changed': [
+        event: {
+            code: 'properties';
+            value: ValueOf<Schemas['ProductListingResult']['currentFilters']['properties']>;
+        },
+    ];
+}>();
+
+// TODO selected values filter und nur die drin lassen, die auch in filter.options drin sind
```
Comment (m-borgmann) [inline components/ProductListing/Filter/Options/ProductListingFilterOptionsProperties.vue:21]: Are this comment and the one further below supposed to stay?

**Review by m-borgmann — APPROVED** [review]

File: components/ProductListing/Filter/Options/ProductListingFilterOptionsPrice.vue [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<script setup lang="ts">
+import type { ListingFilter } from '~/types/listing/filter';
```
Comment (m-borgmann) [inline components/ProductListing/Filter/Options/ProductListingFilterOptionsPrice.vue:2]: Can we have auto importing for the types? especially the valueOf import is in a bunch of files in this PR

Comment (mloeffle) [reply components/ProductListing/Filter/Options/ProductListingFilterOptionsPrice.vue]: Types should now be auto imported.

**Review by m-borgmann — APPROVED** [review]

**Pull request merged**